app.js

```diff
@@ -340,6 +340,61 @@ var comfyJS = {
     }
     return false;
   },
+  Announce: function( message, channel ) {
+    if( client ) {
+      if( !channel ) {
+        channel = mainChannel;
+      }
+      client.say( channel, `/announce ${message}` )
+      .catch( comfyJS.onError );
+      return true;
+    }
+    return false;
+  },
+  AnnounceBlue: function( message, channel ) {
```

Comment:
I think this is better as an option in the parameter like `Announce: function( message, channel, color = null )` so that it can be passed in. What do you think?

Reply:
Yeh I wanted to refactor it
maybe add the color to Announce
and then let announceblue call announce with that color